assets/src/scss/new-identity/style.scss

```diff
@@ -76,6 +76,18 @@ table.spreadsheet-table.is-color-dark-green {
   }
 }
 
+a:where(.standalone-link) {
+  width: fit-content;
+  border-bottom: 2px solid var(--gp-green-500);
+  font-weight: 700 !important;
+  transition: 0.3s cubic-bezier(0.5, 0, 0, 1);
+
+  &:hover {
+    box-shadow: inset 0 calc(-22px) var(--gp-green-100);
+    text-decoration: none !important;
```

Comment:
This is because of [this](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/base/_css-variables.scss#L9)

Reply:
Since most links will have this style, I think it would be better to override the `--link--hover--text-decoration` to `none` and add new variables for the links (such as contextual links) that still need the underline on hover... But we can also think about that in the PR for body links!